### ISSUE TITLE
Upgrade Akka-Typed sample to the Akka 2.5.x based?

### ISSUE THREAD

**Issue by He-Pin** [ReactiveDesignPatterns/CodeSamples#13]
I know the book is using Akka 2.4.x，but yes,there is something changed.

**Comment by He-Pin** [comment]
chapter 15 needs an overhaul
https://github.com/akka/akka-meta/issues/62

**Comment by rkuhn** [comment]
Yes, that is true, especially since 2.4 has reached EOL. Tonight I’m out of steam, but this will be a good thing to dabble with when I get a free evening.

**Comment by He-Pin** [comment]
@rkuhn How about the book?will there be an update there, I mean in chapter 15.
